File: app/src/prod/java/br/com/githubrepos/data/endpoint/PullRequestEndpoint.java
package br.com.githubrepos.data.endpoint;

import java.util.List;

import br.com.githubrepos.data.entity.PullRequest;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface PullRequestEndpoint {

    @GET("/repos/{owner}/{repository}/pulls")
    Call<List<PullRequest>> list(@Path("owner") String owner, @Path("repository") String repository);
}

File: app/src/main/java/br/com/repository/interfaces/endpoint/PullRequestEndPoint.kt
package br.com.repository.interfaces.endpoint

import br.com.repository.model.PullRequest
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

interface PullRequestEndPoint {

    interface Pull {
        @GET("repos/{login}/{name}/pulls")
        fun callPullRequest(@Path("login") creator: String, @Path("name") name: String): Observable<List<PullRequest>>
    }


}